app/backend/src/tests/test_communities.py

```diff
@@ -0,0 +1,467 @@
+            assert lists_equal(res.admin_user_ids, [user4_id, user5_id])
+
+    def test_ListMembers(testing_communities):
+        with session_scope() as session:
+            user1_id, token1 = get_user_id_and_token(session, "user1")
+            user2_id, token2 = get_user_id_and_token(session, "user2")
+            user3_id, token3 = get_user_id_and_token(session, "user3")
+            user4_id, token4 = get_user_id_and_token(session, "user4")
+            user5_id, token5 = get_user_id_and_token(session, "user5")
+            user6_id, token6 = get_user_id_and_token(session, "user6")
+            user7_id, token7 = get_user_id_and_token(session, "user7")
+            user8_id, token8 = get_user_id_and_token(session, "user8")
+            w_id = get_community_id(session, "World")
+            c1r1c2_id = get_community_id(session, "Country 1, Region 1, City 2")
+
+        with communities_session(token1) as api:
+            res = api.ListMembers(
+                communities_pb2.ListMembersReq(
+                    community_id=w_id,
+                )
+            )
+            assert lists_equal(
+                res.member_user_ids, [user1_id, user2_id, user3_id, user4_id, user5_id, user6_id, user7_id, user8_id]
+            )
+
+            res = api.ListMembers(
+                communities_pb2.ListMembersReq(
+                    community_id=c1r1c2_id,
+                )
+            )
+            assert lists_equal(res.member_user_ids, [user2_id, user4_id, user5_id])
+
+    def test_ListNearbyUsers(testing_communities):
+        with session_scope() as session:
+            user1_id, token1 = get_user_id_and_token(session, "user1")
+            user2_id, token2 = get_user_id_and_token(session, "user2")
+            user3_id, token3 = get_user_id_and_token(session, "user3")
+            user4_id, token4 = get_user_id_and_token(session, "user4")
+            user5_id, token5 = get_user_id_and_token(session, "user5")
+            user6_id, token6 = get_user_id_and_token(session, "user6")
+            user7_id, token7 = get_user_id_and_token(session, "user7")
+            user8_id, token8 = get_user_id_and_token(session, "user8")
+            w_id = get_community_id(session, "World")
+            c1r1c2_id = get_community_id(session, "Country 1, Region 1, City 2")
+
+        with communities_session(token1) as api:
+            res = api.ListNearbyUsers(
+                communities_pb2.ListNearbyUsersReq(
+                    community_id=w_id,
+                )
+            )
+            assert lists_equal(
+                res.nearby_user_ids, [user1_id, user2_id, user3_id, user4_id, user5_id, user6_id, user7_id, user8_id]
+            )
+
+            res = api.ListNearbyUsers(
+                communities_pb2.ListNearbyUsersReq(
+                    community_id=c1r1c2_id,
+                )
+            )
+            assert lists_equal(res.nearby_user_ids, [user4_id])
+
+
+# TODO: requires transferring of content
```

Comment:
I'm usually against leaving TODO:s as comments in the code, I rather would like an issue opened. But I don't feel strongly about it so I'm ok with leaving it as-is.

Reply:
Yes, I was going to work on those next! (Implementation then tests)